logging/dedicated-efk-logging.adoc

```diff
@@ -0,0 +1,65 @@
+:context: dedicated-efk-logging
+[id="dedicated-efk-logging"]
+= About cluster logging and {product-title}
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+As an {product-title} cluster administrator, you can deploy cluster logging
+to aggregate logs for a range of services.
+
+{product-title} clusters come configured with logging via `ElasticSearch`
+operators installed and `Curator` pre-installed to have a two-day retention of
+logs. To utilize these features, create a `ClusterLogging` Custom Resource in
+the `openshift-logging` namespace with the following `.yaml` information:
+
+----
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogging"
+metadata:
+  name: "instance"
+  namespace: "openshift-logging"
+spec:
+  managementState: "Managed"
+  logStore:
+    type: "elasticsearch"
+    elasticsearch:
+      nodeCount: 3
+      storage: {}
+      redundancyPolicy: "SingleRedundancy"
+      nodeSelector:
+        node-role.kubernetes.io/worker: ""
+      resources:
+        request:
+          memory: 8G
+  visualization:
+    type: "kibana"
+    kibana:
+      replicas: 1
+      nodeSelector:
+        node-role.kubernetes.io/worker: ""
+  curation:
+    type: "curator"
+    curator:
+      schedule: "30 3 * * *"
+      nodeSelector:
+        node-role.kubernetes.io/worker: ""
+  collection:
+    logs:
+      type: "fluentd"
+      fluentd: {}
+      nodeSelector:
+        node-role.kubernetes.io/worker: ""
+----
+
+[NOTE]
+====
+To opt-out of logging after it has been installed, delete the `ClusterLogging`
```

Comment:
s/opt-out/opt out
opt out is the verb form.

But isn't logging already installed? I might say "To opt out of logging after you configure it, delete the `ClusterLogging` custom resource that you created."

Reply:
Yep, I agree. Fixed!